### ISSUE TITLE
[Content styleguide] Content, entry points, and link labels don't use plain language or familiar words, terms, and acronyms that users understand. (05.01.3)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#68710]
#### VFS team name
Contention Classification

#### VFS product name
Disability Comp Application (21-526EZ)

#### VFS feature name
update suggested conditions list

#### Point of Contact/Reviewers
Brian Smolensky - @BSmolensky - Content

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Content, entry points, and link labels don't use plain language or familiar words, terms, and acronyms that users understand.

### Issue Details
At the bottom of the log-in page, the user finds the words "Respondent burden" and a time next to it. "Respondent Burden" is not plain language. Please reword to allow all users to understand the meaning of the time listed. 

### VA.gov Experience Standard
[Category Number 05, Issue Number 01](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.1.4 AAA

### Platform Recommendation
Please put in plain language. 

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by RakshindaAslam** [comment]
@christinec-fftc - is this part of the OMB component?

**Comment by RakshindaAslam** [comment]
Need to check what is displayed on other forms on va.gov

**Comment by christinec-fftc** [comment]
This text is coming from the component library in the [omb-info component](https://design.va.gov/components/omb-info), so not specific to 526EZ.

**Comment by shiragoodman** [comment]
FYI @caw310 @micahtaylor . Adding the DST label to this ticket.

**Comment by caw310** [comment]
@micahchiang I think Shira meant to tag you on this.

**Comment by micahchiang** [comment]
"Respondent Burden" has been used for a long time, and it's the language used on VA paper forms [like this 526 pdf.](https://www.vba.va.gov/pubs/forms/vba-21-526ez-are.pdf) (top right corner page 9). Obviously, those things don't necessarily make it right to use, but this is the first time I've heard of it being an issue. Have there been any conversations going on in Slack that can provide some additional context about this? 

@humancompanion-usds - any thoughts on this? 

Technically, the change is trivial, just want to be sure this is a decision the right parties are agreeing on.
